packages/backend/src/apps/telegram-bot/actions/send-message/index.ts

```diff
@@ -47,6 +47,15 @@ const action: IRawAction = {
       description:
         'Sends the message silently. Users will receive a notification with no sound.',
     },
+    {
+      label: 'Topic ID',
+      key: 'topicId',
+      type: 'string' as const,
```

Comment:
Can we time box 1 hour to convert this to allowArbitrary dropdown, as discussed.

Thanks!